activesupport/lib/active_support/encrypted_file.rb

```diff
@@ -26,18 +26,20 @@ def self.generate_key
     end
 
 
-    attr_reader :content_path, :key_path, :env_key
+    attr_reader :content_path, :key_path, :env_key, :raise_if_missing_key
 
-    def initialize(content_path:, key_path:, env_key:)
+    def initialize(content_path:, key_path:, env_key:, raise_if_missing_key:)
       @content_path, @key_path = Pathname.new(content_path), Pathname.new(key_path)
-      @env_key = env_key
+      @env_key, @raise_if_missing_key = env_key, raise_if_missing_key
```

Comment:
Can you make this two lines? This causes an array to be allocated where one isn't needed. So instead of 

```ruby
@env_key, @raise_if_missing_key = env_key, raise_if_missing_key
```

I would

```ruby
@raise_if_missing_key = raise_if_missing_key
@env_key = env_key